sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/MultiMasterConflictResolutionTest.java

```diff
@@ -81,7 +81,7 @@ public void conflictResolutionPolicyCRUD() {
             // when (e.StatusCode == HttpStatusCode.BadRequest)
             CosmosException dce = Utils.as(e, CosmosException.class);
             if (dce != null && dce.getStatusCode() == 400) {
-                assertThat(dce.getMessage()).contains("Invalid path '\\/a\\/b' for last writer wins conflict resolution");
+                assertThat(dce.getMessage()).contains("Invalid path '\\\\\\/a\\\\\\/b' for last writer wins conflict resolution");
```

Comment:
is this because of service side message change?

Reply:
Yes, that is the reason :)